docs/changelog-fragments.d/786.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+OpenSSL system call errors are now intercepted and reraised as standard Python ConnectionErrors.
```

Comment:
Since we have the upstream CPython docs plugged via intersphinx, we can make them clickable.
```suggestion
OpenSSL system call errors are now intercepted and reraised as standard Python :exc:`ConnectionErrors`.
```